office/pdftk/pdftk.SlackBuild

```diff
@@ -51,49 +43,38 @@ TMP=${TMP:-/tmp/SBo}
 PKG=$TMP/package-$PRGNAM
 OUTPUT=${OUTPUT:-/tmp}
 
-if [ "$ARCH" = "i586" ]; then
-  SLKCFLAGS="-O2 -march=i586 -mtune=i686"
-  LIBDIRSUFFIX=""
-elif [ "$ARCH" = "i686" ]; then
-  SLKCFLAGS="-O2 -march=i686 -mtune=i686"
-  LIBDIRSUFFIX=""
-elif [ "$ARCH" = "x86_64" ]; then
-  SLKCFLAGS="-O2 -fPIC"
-  LIBDIRSUFFIX="64"
-else
-  SLKCFLAGS="-O2"
-  LIBDIRSUFFIX=""
-fi
-
 set -e
 
 rm -rf $PKG
 mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
-rm -rf $PRGNAM-$VERSION-dist
-unzip $CWD/$PRGNAM-$VERSION-src.zip
-cd $PRGNAM-$VERSION-dist
+
 chown -R root:root .
 find -L . \
  \( -perm 777 -o -perm 775 -o -perm 750 -o -perm 711 -o -perm 555 \
   -o -perm 511 \) -exec chmod 755 {} \; -o \
  \( -perm 666 -o -perm 664 -o -perm 640 -o -perm 600 -o -perm 444 \
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \;
 
-patch -p1 < $CWD/pdftk-2.02-use-gcc5.patch
+# Copy the binary
+mkdir -p $PKG/usr/share/java/$PRGNAM
+cp $CWD/$PRGNAM-all.jar $PKG/usr/share/java/$PRGNAM
```

Comment:
All the steps in this slackbuild doing `cp $CWD/...` should use `install` instead imho so you can set the mode and owner/group explicitly.

Reply:
You mean the following?
install -Dm644 $CWD/$PRGNAM-all.jar $PKG/usr/share/java/$PRGNAM
install -Dm755 $CWD/$PRGNAM.sh $PKG/usr/bin/$PRGNAM
install -Dm644 $CWD/$PRGNAM.1 $PKG/usr/man/man1

Reply:
Yes, exactly